tfjs-backend-webgpu/src/kernels/reduce_webgpu.ts

```diff
@@ -30,7 +30,7 @@ export class ReduceProgram implements WebGPUProgram {
   workGroupSize: [number, number, number];
   variableNames = ['x'];
   uniforms = 'int reduceSize;';
-  uniformsWgsl = 'reduceSize : u32;';
+  uniformsWgsl = 'reduceSize : i32;';
```

Comment:
It's not necessary to change `reduceSize` type from `u32` to `i32`？
And it's also weird to do below change:
```
       fn DIV_CEIL(a : u32, b : u32) -> u32 {
        return u32(i32(a) - 1) / b + 1u;
       }
```
if a = 0u, in anyway, it is not correct unless you change a/b's type to `i32`